en/developer-docs/docs/consuming-services/generate-an-access-token.md

```diff
@@ -1,4 +1,4 @@
-# Generate an Access Token
+# Generate a1. An application in the [Choreo Developer Portal](https://devportal.choreo.dev). If you don't have one, [create a new application](https://wso2.com/choreo/docs/consuming-services/manage-application/#step-1-create-an-application). Access Token
```

Comment:
There is a typo in this heading. Let's fix it.

Reply:
Fixed in eb419ef0adc0e08c6394003fb08826f73180a7df